compose/cli/docopt_command.py

```diff
@@ -34,6 +34,10 @@ def parse(self, argv):
         if docstring is None:
             raise NoSuchCommand(command, self)
 
+        # checks to see if the command is help and if so outputs top level command options
+        if command == 'help' and options['ARGS'] == []:
+            raise SystemExit(docstring + '\n-------------------\n\n' + command_help)
```

Comment:
Interesting comment and question.  I would not consider myself a python developer.  Just a guy interested in learning Docker and Python and this seemed like a great way to dive in.  Now to your question.  In docopt_command.py I followed the pattern that was being used within that script.  If no command was provided an exception was raised to output the top level commands. If a docstring for the command was not provided an exception was raised.  Likewise I thought if the help command was not used with out a command such as 'docker-compose help run' or 'docker-compose help up' then the top level commands would be outputted to stdout since the help command is intended to be used with a command option and not by itself.

If this is pattern is not ideal or doesn't follow python best practices, might be a good task to place in the list for future clean up.  Thoughts?